.github/workflows/OCV-Coverage-4.x-U20.yaml

```diff
@@ -0,0 +1,295 @@
+    - name: Accuracy:photo
+      timeout-minutes: 240
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && xvfb-run -a bin/opencv_test_photo --test_threads=2
+    - name: Accuracy:stitching
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && xvfb-run -a bin/opencv_test_stitching --test_threads=2
+    - name: Accuracy:video
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && xvfb-run -a bin/opencv_test_video --test_threads=2
+    - name: Accuracy:videoio
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && xvfb-run -a bin/opencv_test_videoio --test_threads=2
+    - name: Python3
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: |
+        cd ${{ env.OPENCV_DOCKER_WORKDIR }}/modules/python/test
+        python3 ./test.py --repo ../../../ -v
+    - name: Java
+      timeout-minutes: 60
+      id: java-test
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && xvfb-run -a python3 ${{ env.OPENCV_DOCKER_WORKDIR }}/modules/ts/misc/run.py . -a -t java --test_threads=2
+    - name: Save Unit Test Results
+      timeout-minutes: 60
+      uses: actions/upload-artifact@v3
+      if: ${{ always() && steps.java-test.outcome == 'success' && steps.build-opencv.outcome == 'success' }}
+      with:
+        name: junit-html
+        path: /home/ci/build/java_test/testResults/junit-noframes.html
+    - name: Run test coverage-aggregation
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && lcov --directory /home/ci/build --capture -o /home/ci/build/opencv_test.info
+    - name: Combine test coverage
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && lcov -a /home/ci/build/opencv_test_base.info -a /home/ci/build/opencv_test.info -o /home/ci/build/opencv_test_total.info
+    - name: Filter test coverage
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && lcov --remove /home/ci/build/opencv_test_total.info '/usr/include/*' '/usr/lib/*' '/usr/local/include/*' '*/opencv/apps/*' '*/opencv/samples/*' '*/opencv/3rdparty/*' '*/opencv/modules/ts/*' '*/opencv/modules/*/perf/*' '*/opencv/modules/*/test/*' '*/opencv/modules/*/samples/*' '*/home/ci/build/modules/java_bindings_generator/*' '*/home/ci/build/modules/python_bindings_generator/*' '*/opencv_contrib/modules/*/perf/*' '*/opencv_contrib/modules/*/test/*' '*/opencv_contrib/modules/*/samples/*' '*/opencv_contrib/*' -o /home/ci/build/opencv_test_filtered.info
```

Comment:
Need to filter out `build/*` entries too (like `build/3rdparty/ade/ade-0.1.1f/sources/ade/include/ade`)

Also, probably "Release" build inlines some functions. The difference from [this build](https://pullrequest.opencv.org/buildbot/export/opencv_releases/master_coverage-lin64-debug/20220610-175221_100000/coverage_html/index.html):

```.diff
-opencv/modules/core/src	75.4% 75.4% 75.4 %	23139 / 30670	63.7 %	2613 / 4105
+opencv/modules/core/src	75.4% 75.4% 75.4 %	19995 / 26515	63.3 %	2487 / 3930
```

Reply:
Added `build/*` also.

I've seen some issues in "Debug" build, tests fail with `Segmentation fault (core dumped)` [error](https://github.com/opencv/ci-gha-workflow/runs/6895113119?check_suite_focus=true#step:50:636).

Reply:
It is better to investigate and debug this crash somehow on build machines (instead of switching to Release mode).

Reply:
Got your point. I'll debug it.

Reply:
Fixed the issue. At the moment, it's in the debug mode and no `Segmentation fault (core dumped)` errors.

Reply:
In addition, I've changed the cron time for this job to run it weekly at 12am (UTC) on Saturday, because coverage pipeline is running ~12h.

Reply:
> ~12h

BuildBot-based [configuration](https://pullrequest.opencv.org/buildbot/builders/master_coverage-lin64-debug/builds/100000) requires ~2h.

Try to apply `--test_tag_skip=size_hd` and compare coverage.